x-pack/plugins/apm/scripts/bootstrap_apm/src/run.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -u
+scriptdir="$(dirname "$0")"
+cd "$scriptdir"
+source variables.sh
+source utils/logger.sh
+
+if ! [ -x "$(command -v ecctl)" ]; then
+  echo -e "${RED}ERROR: ${NC}ecctl not found.${NC}"
```

Comment:
Are `RED` and `NC` osx default things?. I was expecting to see something like below if your variables:

```
RED='\033[0;31m'
NC='\033[0m' 
```

Perhaps I'm missing something? :)